bundles/net.tourbook/src/net/tourbook/Messages.java

```diff
@@ -483,7 +497,6 @@
    public static String        app_action_open_tour;
    public static String        app_action_quick_edit;
    public static String        app_action_read_edit_tooltip;
-   public static String        app_action_save;
```

Comment:
@wolfgang-ch 
I removed that one because when I looked at the translation files it seemed to be a duplicate of [that one](https://github.com/FJBDev/mytourbook/blob/df98402e4bb417c74301276a260e92439911ea94/bundles/net.tourbook/src/net/tourbook/Messages.java#L546)  `App_Action_Save`

Reply:
👍